src/NHSD.BuyingCatalogue.Database.PublishAll/README.md

```diff
@@ -0,0 +1,56 @@
+# Data Migration Package
+
+This is a SSIS package migrates the following tables from one buying catalogue database to another.
+
+* CatalogueItem
+* Solution
+* AssociatedService
+* AdditionalService
+* FrameworkSolutions
+* MarketingContact
+* SolutionCapability
+* SolutionEpic
+* SupplierServiceAssociation
+
+## Pre-requisites
+
+* Visual Studio 2019
```

Comment:
I assume we don't need a specific version of the .Net Framework?

Reply:
I've done a bit of hunting around and I'm struggling to find anywhere that Microsoft lists a specific .NET Framework requirement.

.NET is required for SSIS but I can only assume that the installation steps outlined in the readme must make the sure the necessary framework is present and/or install if not.

Reply:
I think the .Net Framework is installed as part of Visual Studio anyway. So what you've outlined is fine. See how the team get on once this is merged in.